pom.xml

```
@@ -22,8 +22,9 @@
     <properties>
         <revision>1.38</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.150.1</jenkins.version>
+        <jenkins.version>2.249.3</jenkins.version>
```

One thought was to bump up to 2.277.4 since that is when `ssh-slaves` is included in the bom.

---

I was thinking go further, as there to not have any detached dependencies 😀

---

How far would be your preference?
Durable-task actually starts having termination issues with `Big Sur` due to some implementation bugs in `ProcessTree`.  See https://github.com/jenkinsci/jenkins/pull/5548 can we go bleeding edge? 😄

---

```suggestion
        <jenkins.version>2.249.1</jenkins.version>
```

acc. to https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/

Fine from my PoV to go with 2.289.x or whatever if there are actually problems with older lines that affect development & testing.

---

I settled for 2.277 since the largest grouping of users live on 2.277.3 and 4. Once there is an LTS that  includes https://github.com/jenkinsci/jenkins/pull/5548 I'm going to move to it, so might as well start moving  people along.

393fcc5eb12afdb0f08dd39e4813c4433a254678